benchexec/tools/swat.py

```diff
@@ -0,0 +1,45 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+from benchexec.tools.template import BaseTool2
+
+
+class Tool(BaseTool2):
+    """
+    Tool info module for SWAT, a dynamic symbolic execution tool for Java 17.
+    SWAT is currently being developed by the Institute for IT Security at the University of Luebeck ( https://www.its.uni-luebeck.de/en/institute.html ).
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("run-swat.sh")
+
+#    def version(self, executable):
+#        return '42'
+#        #return self._version_from_tool(executable, arg="-v")
+
```

Comment:
Just remove this for now. The line with `return '42'` does not make sense at all, and the whole comment does not conform to our code formatter, so would need to be reformatted otherwise.